File: web/enums/chat/AssistantActivityEnum.ts
// TODO: status pills shown while the assistant works through a request - move to server/database
enum AssistantActivityEnum {
  PowerCheck = 'power_check',
  ManualLookup = 'manual_lookup',
  ReviewLog = 'review_log',
  SystemCheck = 'system_check',
  WeatherCheck = 'weather_check',
}

export default AssistantActivityEnum;

File: api/enums/chat/assistant_activity_enum.py
from enum import StrEnum


class AssistantActivityEnum(StrEnum):
    POWER_CHECK = "power_check"
    MANUAL_LOOKUP = "manual_lookup"
    REVIEW_LOG = "review_log"
    SYSTEM_CHECK = "system_check"
    WEATHER_CHECK = "weather_check"
